tests/fixtures/closing-of-element-with-open-child-elements/expected/x/component/component.html

```diff
@@ -0,0 +1,5 @@
+<template>
+    <section>
+        <span>hello</span>
+        <span>mr. anderson</span>
+</section></template>
```

Comment:
There isn't an easy way to reformat the tags so they line up perfectly with their starting tag.